website/pages/en/quick-start.mdx

```diff
@@ -2,24 +2,20 @@
 title: Quick Start
 ---
 
-This guide will quickly take you through how to initialize, create, and deploy your subgraph to Subgraph Studio.
+This guide walks you through the basics of creating, deploying, publishing, and querying a subgraph on The Graph easily.
```

Comment:
wondering that if the word `subgraph` should link to some glossary which explains that a Subgraph is for the folks who are completely new this is their quick start tutorial and not everyone would know what a subgraph is.

Reply:
Yes, agree!

Reply:
Maybe here: https://thegraph.com/docs/en/developing/developer-faqs/#1-what-is-a-subgraph